bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/handler/NanoleafControllerHandler.java

```diff
@@ -466,12 +629,16 @@ private void updateFromControllerInfo() throws NanoleafException, NanoleafUnauth
                 panelHandler.updatePanelColorChannel();
             }
         });
+
+        @Nullable Layout layout = controllerInfo.getPanelLayout().getLayout();
+        String layoutView = (layout!=null) ? layout.getLayoutView(): "";
+        logger.info("Panel layout and ids for controller {} \n {}", thing.getUID(), layoutView);

```

Comment:
Rather not use newlines in logs and do not use info logging for such (debug) information